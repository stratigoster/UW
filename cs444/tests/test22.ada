PACKAGE foo IS
  TYPE data IS RECORD
    null;
    null;
--  ^^^^^
-- null record can only contain one declaration: `null;'
  END RECORD;
END foo;
