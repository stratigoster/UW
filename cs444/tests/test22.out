***ERROR***:4:5:	Expected "end" but found "null"
